Source/Core/DiscIO/VolumeWad.h

```diff
@@ -14,10 +14,6 @@
 #include "Core/IOS/ES/Formats.h"
 #include "DiscIO/Volume.h"
 
-// --- this volume type is used for Wad files ---
```

Comment:
You should either keep this line or remove the equivalent lines in VolumeWii and VolumeGC. (Probably the latter.)

Reply:
Removed them.